project/Dependencies.scala

```diff
@@ -203,6 +203,8 @@ object Dependencies {
       exclude("com.google.cloud.datastore", "datastore-v1-protos")
       exclude("org.apache.httpcomponents", "httpclient"),
     "com.google.cloud" % "google-cloud-compute" % googleCloudComputeV,
+    "org.broadinstitute.dsde.workbench" %% "workbench-google" % "0.10-0967a99"
```

Comment:
Yeah, I forgot about that in the interim. the upside is this has hung around so long that I no longer need to make a PR to workbench-libs like I Said I would.

re the constant: it's only used in one place, IMO string constants should only be used when n > 1

Reply:
Yeah, I get you on constants when n == 1. Still a trip through git history shows our habit of copy-pasting and then _not_ creating the constant when going to n == 2, n == 3, etc. The worst is when different versions of the same library to two different sub-projects.

Also having the constants in one block, alphabetically, makes it easier to update when running `sbt dependencyUpdates`. I suspect with our large number of libraries we'll have at least some patching to do soon enough after security audits, etc. (ask GOTC about their recent cleanups)

Reply:
My answer to your first point about poor history is to fix that poor behavior ;)

However you won me over with your second paragraph. Particularly considering I've had [this](https://github.com/rtimush/sbt-updates) open as a tab in my browser for a while because I get so excited about dependency updates :)